app/src/main/java/com/rob729/quiethours/Fragments/MainFragment.kt

```diff
@@ -165,42 +168,27 @@ class MainFragment : Fragment() {
         return when (item.itemId) {
             R.id.action_settings -> {
                 return view?.let { Navigation.findNavController(it) }?.let {
-                    NavigationUI.onNavDestinationSelected(
-                        item,
-                        it
-                    )
+                    NavigationUI.onNavDestinationSelected(item, it)
                 }!! || super.onOptionsItemSelected(item)
             }
             R.id.action_delete -> {
                 if (binding.emrl.visibility == View.GONE) {
                     AlertDialog.Builder(requireContext())
-                        .setTitle("Delete All Profiles")
-                        .setMessage("Are you sure you want to delete all the profiles?")
+                        .setTitle("Delete All Profiles").setMessage("Are you sure you want to delete all the profiles?")
```

Comment:
This is not the expected solution. Let it remain as it was for better readability.